examples/cfq/train.py

```diff
@@ -80,56 +80,62 @@ def create_model(vocab_size: int) -> nn.Module:
 
 
 def cross_entropy_loss(logits: jnp.array, labels: jnp.array,
-                       lengths: jnp.array):
+                       lengths: jnp.array, vocab_size: int):
   """Returns cross-entropy loss."""
+  labels = common_utils.onehot(labels, vocab_size)
   log_soft = nn.log_softmax(logits)
   log_sum = jnp.sum(log_soft * labels, axis=-1)
   masked_log_sum = jnp.mean(mask_sequences(log_sum, lengths))
   return -masked_log_sum
 
 
-def pad_batch_to_max(batch: jnp.array, seq_len: int, max_len: int):
-  """Pads the input array on the 2nd dimension to the given length
-    (padding is done with 0)
-
-    Args:
-      batch: batch array
-      seq_len: 2nd dimension current value
-      max_len: 2nd dimension desired value
-    """
-  padding_size = max_len - seq_len
-  padding = tf.constant([[0, 0], [0, padding_size], [0, 0]])
-  return tf.pad(batch, padding, "CONSTANT").numpy()
+def pad_along_axis(array: jnp.array,
+                   padding_size: int,
+                   axis: int) -> jnp.array:
+  """Returns array padded with zeroes on given axis with padding_size positions.
+  The padding is done at the end of the array at that axis"""
+  # The pad function expects a shape of size (n,2) where n is the number of axes
+  # For each axis, the number of padding positions at the begining and end of
+  # the array is specified
+  pad_shape = jnp.full((array.ndim, 2), 0)
```

Comment:
Is there are reason you are not using `jnp.zeros` here, like I suggested?

```suggestion
  pad_shape = jnp.zeros((array.ndim, 2))
```

Reply:
no